unblob/handlers/filesystem/fat.py

```diff
@@ -132,10 +149,72 @@ def valid_header(self, header: Instance) -> bool:
             return self.valid_fat32_header(header)
         return True
 
+    def get_fat_size(self, header: Instance) -> int:
```

Comment:
the name suggests it returns the size, but in reality it returns the sector count, either return real size, or change function name